frontend/src/metabase/collections/components/PinnedItems.jsx

```diff
@@ -30,12 +30,19 @@ const PinnedItem = ({ item, index, collection, onCopy, onMove }) => (
       onPin={() => item.setPinned(false)}
       onMove={onMove}
       onCopy={onCopy}
+      selectable={false}
```

Comment:
We actually want pinned items to be selectable for bulk actions. It was broken and fixed recently, here is the related issue: #15338

Reply:
Oh, good catch! I thought it was intended only for unpinned items and we just forgot to hide the checkbox. I'll fix that, thanks!

Reply:
I'm also worried that the repro test didn't fail 🤔 Can you give it a look please?